doc/changes/changelog.md

```diff
@@ -1,4 +1,5 @@
 # Changes
 
+* [11.1.0](changes-11.1.0.md)
```

Comment:
is it a minor release?
Since you also removed capabilities it breaks API, doesn't it?

Reply:
It's a feature release. I only added a few new capabilities, I didn't remove anything